src/Sarif.FunctionalTests/v2/ConverterTestData/PREfast/Expected-101.xml.sarif

```diff
@@ -12,6 +12,24 @@
           "mimeType": "text/x-cpp"
         }
       },
+      "logicalLocations": {
+        "bad1": {
+          "name": "bad1",
+          "decoratedName": "?bad1@@YAXPAHIPAU_FLEXARRAY@@@Z"
+        },
+        "bad2": {
+          "name": "bad2",
+          "decoratedName": "?bad2@@YAPAU_FLEXARRAY@@I@Z"
+        },
+        "Foo::what": {
+          "name": "Foo::what",
```

Comment:
Just to emphasize: Here `name` is required, because it should be `"what"`, which doesn't equal the key `"Foo::what"`.